### ISSUE TITLE
Making excerpts editable

### ISSUE THREAD

**Issue by kynan** [hypothesis/browser-extension#38]
It would be great to be able to added the excerpt when making annotations.

**Comment by robertknight** [comment]
Are there any particular scenarios where you find yourself wanting to edit the excerpt?

**Comment by kynan** [comment]
Yes, one particular case the excerpt included lots of weird formatting, including JavaScript etc.

**Comment by tophee** [comment]
Yes, in my experience, it happens quite frequently in pdf documents that the excerpt is not accurate (mostly missing a couple of letters at the end). Assuming that all pdf files will never be faultless, the best way to handle this is to allow users to at least fix the excerpt.
